assets/js/worker.js

```diff
@@ -0,0 +1,26 @@
+onmessage = function(e) {
+  importScripts('vendor/lunr.min.js')
+
+  var docs = JSON.parse(e.data);
+
+  lunr.tokenizer.separator = /[\s/]+/
```

Comment:
why remove `{{ site.search.tokenizer_separator | default: site.search_tokenizer_separator | default: "/[\s\-/]+/" }}`?

we can also submit this PR to upstream, but then we'd have to keep all features supported.

Reply:
Good catch! I was just editing the generated JS for quick testing and forgot to re-apply this.